code/modules/mob/living/carbon/xenomorph/castes/crusher/abilities_crusher.dm

```diff
@@ -105,6 +105,9 @@
 		for(var/obj/effect/forcefield/fog/fog in throw_origin)
 			A.balloon_alert(X, "Cannot, fog")
 			return fail_activate()
+	if(isvehicle(A))
```

Comment:
this is checking for ALL vehicles... i.e. a wheelchair. Check for armoured or whatever the istype is for these specific types of vehicles.

Reply:
Alright, done. I thought we didn't have one for apc/tank, but turns out we do.